Uplift/Views/ClassDetail/ClassDetailHeaderView.swift

```diff
@@ -99,6 +145,31 @@ class ClassDetailHeaderView: UICollectionReusableView {
         let locationButtonHeight = 16
         let semicircleImageViewSize = CGSize(width: 100, height: 50)
         let titleLabelHorizontalPadding = 40
+        let backButtonLeftPadding = 20
+        let backButtonSize = CGSize(width: 24, height: 24)
+        let backButtonTopPadding = 47
+        let favoriteButtonRightPadding = 20
+        let favoriteButtonSize = CGSize(width: 23, height: 23)
+        let shareButtonRightPadding = 14
+        let shareButtonSize = CGSize(width: 24, height: 24)
```

Comment:
can we generalize these constants? ex `backButtonSize` and `shareButtonSize` are the same, can they just become `buttonSize`?